recipes/cosap/meta.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: cosap
+  version: 0.1.0
+
+source:
+  url: https://github.com/MBaysanLab/cosap/archive/refs/tags/v0.1.0.tar.gz
+  sha256: 866b0335360f30f5603c0775d3d7072a0987ab0e269950e857557dac79a9dc0a
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - cosap=cosap._cosap:cosap_cli
+  script: "python -m pip install . -vv"
+  run_exports:
+    - {{ pin_subpackage("cosap", max_pin='x.x') }}
+
+requirements:
+  build:
+    - python ==3.9.15
+    - setuptools
+    - pip
+  run:
+    - openjdk ==17.0.3
```

Comment:
Does it need to use exactly 17.0.3 ?
I guess any version newer than 17.x will be fine ?
I'd suggest to remove all the version pinning below and set `>=x.y.z` only where really needed. This will make upgrades easier!

Reply:
Ping!

Reply:
@martin-g Thanks for the review and sorry for the delay. We've made the some of the version pinnings less strict and removed some. But we still want to keep other package versions as they are for reproducibility reasons.